src/main/java/net/minestom/server/instance/LightingChunk.java

```diff
@@ -107,7 +108,15 @@ private boolean checkSkyOcclusion(Block block) {
         return occludesBottom || occludesTop;
     }
 
-    private void invalidateSection(int coordinate) {
+    public void setFreezeInvalidation(boolean freezeInvalidation) {
+        this.freezeInvalidation = freezeInvalidation;
+    }
+
+    public void invalidateSection(int coordinate) {
+        if (freezeInvalidation) {
```

Comment:
Why to do check here if this one already is in `setBlock`? 
Also freeze sounds to me, as it should automatically invalidate all things after being unfreezed.

Reply:
1. because it's now public
2. That's a large side effect for a setter, plus the caller maybe decided to do no changes, but i don,t mind that much